src/connectors/argentMobile/modal/argentModal.ts

```diff
@@ -72,21 +72,10 @@ class ArgentModal {
     const wcParam = encodeURIComponent(wcUri)
     const href = encodeURIComponent(window.location.href)
 
-    /* 
-    // KEEPING IN CASE NEEDS TO BE REVERTED
-    const desktopWcParam = encodeURIComponent(
-      `${this.mobileUrl}app/wc?uri=${wcParam}`,
-
-    this.showModal({
-      desktop: `${this.bridgeUrl}?wc=${desktopWcParam}&device=desktop`,
-      ios: `${this.mobileUrl}app/wc?uri=${wcParam}&href=${href}&device=mobile`,
-      android: `${this.mobileUrl}app/wc?uri=${wcParam}&href=${href}&device=mobile`,
-    })
-
-    ) */
+    const desktopWcParam = encodeURIComponent(wcParam)
```

Comment:
I cannot properly review it without the argent login changes, but that doesn't look correct. it's basically `encodeURIComponent(encodeURIComponent(wcUri))` which doesn't make much sense. 

I the commented out code it was necessary because `desktopWcParam` was an actual link

Reply:
@kamilargent ok, I probably misunderstood last time then

so, in this case, the following should be enough:
```typescript
const wcParam = encodeURIComponent(wcUri)

this.showModal({
      desktop: `${this.bridgeUrl}?wc=${wcParam}&href=${href}&device=desktop`,
      ios: `${this.mobileUrl}app/wc?uri=${wcParam}&href=${href}&device=mobile`,
      android: `${this.mobileUrl}app/wc?uri=${wcParam}&href=${href}&device=mobile`,
    })
```

Reply:
Everything depends on how you treat the `wc` param on the argent login side, when creating the QR code. 

1. If it's just getting encoded into the QR code, then, the previous code should be used (although new `config.mobileUrl` will have to be used as we need here `https;//www.argent.xyz/` link instead of `argent://` for `desktop` flow)
2. If it's just getting the wc param end creates a new qr link, then we are ok with just `const wcParam = encodeURIComponent(wcUri)`
```
const wcParam = getQueryParam(link, "wc")
const href = getQueryParam(link, "href")

const qrCode = "https://www.argent.xyz/app/wc?uri=${wcParam}&href=${href}&device=desktop"
```

Reply:
ok @kamilargent  
I think #2 is ok since the url creation is managed in argent-login

```typescript

// argent-login code
 <FirstConnection
            qrCodeData={`https://www.argent.xyz/app/wc?uri=${walletConnectUrl}`}
          />
```

Reply:
👍 as long as the `walletConnectUrl` is the encoded version of the `wcUri` we should be good